pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -116,7 +116,7 @@ func init() {
 				// BuildControllerFactory.buildDeleteLW
 				{
 					Verbs:     sets.NewString("get", "list", "create", "delete"),
-					Resources: sets.NewString("pods"),
```

Comment:
It's awesome that nodes can do anything to the entire cluster, but
this doesn't make it worse.  I'm shocked we haven't had a critsit to
fix that yet.